data/templates/post-install-photon.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+<%_ if (undefined !== networkDevices) { _%>
+## Set network interfaces
+# Remove old network config
+rm /etc/systemd/network/*
+
+    <%_ networkDevices.forEach(function(d) { _%>
+# Create new network config file for <%=d.device%>
+cat > /etc/systemd/network/<%=d.device%>-static.network << "EOF"
+[Match]
+Name=<%=d.device%>
+
+[Network]
+EOF
+
+chmod 644 /etc/systemd/network/<%=d.device%>-static.network
+
+        <%_ for (conf in d) { _%>
+            <%_ if (undefined === d[conf]) continue; _%>
+            <%_ if ('ipv4' === conf || 'ipv6' === conf) { _%>
+                <%_ ipconfig = d[conf] _%>
+                <%_ cidr = 0 _%>
+                <%_ maskNodes = ipconfig.netmask.match(/(\w+)/g) _%>
+                <%_ maskNodes.forEach(function(i) { _%>
+                    <%_ if ('ipv4' === conf) { _%>
```

Comment:
A NIC can have both ipv4 and ipv6 address, so if both ipv4 & ipv6 is present, you need set two kinds of address to the NIC, but your code only choose the ipv4.